features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,634 @@
+      | ingangsdatum geldigheid (85.10)            | <datumIngangGeldigheid>    |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                                       | waarde                             |
+      | straatnaam (11.10)                         | <straat>                           |
+      | naam openbare ruimte (11.15)               | <straat>                           |
+      | huisnummer (11.20)                         | <huisnummer>                       |
+      | postcode (11.60)                           | <postcode>                         |
+      | woonplaats (11.70)                         | <woonplaats>                       |
+      | identificatiecode verblijfplaats (11.80)   | <adresseerbaarObjectIdentificatie> |
+      | identificatiecode nummeraanduiding (11.90) | <nummeraanduidingIdentificatie>    |
+      | locatiebeschrijving (12.10)                | <locatiebeschrijving>              |
+	  | gemeentecode (92.10)                       | 0599                               |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | <burgerservicenummer>           |
+      | fields              | verblijfplaats.datumVan         |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+      | naam                             | waarde                             |
+      | straat                           | <straat>                           |
+      | huisnummer                       | <huisnummer>                       |
+      | postcode                         | <postcode>                         |
+      | woonplaats                       | <woonplaats>                       |
+      | locatiebeschrijving              | <locatiebeschrijving>              |
+      | adresseerbaarObjectIdentificatie | <adresseerbaarObjectIdentificatie> |
+      | nummeraanduidingIdentificatie    | <nummeraanduidingIdentificatie>    |
+      | datumAanvangAdreshouding         | <datumAanvangAdreshouding>         |
+
+      Voorbeelden:
+      | burgerservicenummer | adressoort          | datumAanvangAdreshouding | straat    | huisnummer | postcode | woonplaats | adresseerbaarObjectIdentificatie | nummeraanduidingIdentificatie | locatiebeschrijving        | datumIngangGeldigheid |
+      | 000000310           | binnenlands adres   | 20150808                 | Boterdiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                            | 20150808              |
+      | 000000322           | locatiebeschrijving | 20000310                 |           |            |          |            |                                  |                               | Woonboot in de Grote Sloot | 20000310              |
+
```

Comment:
Zowel "Boterdiep" als "Woonboot in de grote sloot" aanpassen

Reply:
Aangepast